Update docs for MC-1285 (#144)
hazelcast/management-center-docs: docs/modules/troubleshoot/pages/management-center.adoc

and resume the flow using **Resume**.

NOTE: You may only need to share the information in these logs
if requested by the Hazelcast's support team.

== Experience Timeout

> If you experience a timeout when running scripts or SQL queries, You can increase operation timeout value via setting

Reviewer: ```suggestion
If you experience a timeout when running scripts or SQL queries, you can increase the timeout value 
```

> new value to xref:deploy-manage:system-properties.adoc#hazelcast-mc-cluster-operation-timeout['hazelcast.mc.cluster.operation.timeout.millis'] property.

Reviewer: ```suggestion
in the xref:deploy-manage:system-properties.adoc#hazelcast-mc-cluster-operation-timeout[`hazelcast.mc.cluster.operation.timeout.millis`] property.
```
